codegenerator/cli/src/config_parsing/entity_parsing.rs

```diff
@@ -1198,16 +1362,22 @@ pub enum GqlScalar {
     #[subenum(BuiltInGqlScalar)]
     Boolean,
     #[subenum(AdditionalGqlScalar)]
-    BigInt,
+    BigInt(Option<u32>),
     #[subenum(AdditionalGqlScalar)]
-    BigDecimal,
+    BigDecimal(Option<(u32, u32)>),
```

Comment:
```suggestion
    BigDecimal(Option<(u32, u32)>), //  The optional argument: precision and scale as defined in postgres.
```

Reply:
The tuples are fine but it would be helpful if you made a type alias called `type Precision = u31;` etc and then reference the alias so that what the values represent is annotated in the type.

You could also use a struct.